libs/ledger-live-common/src/families/elrond/__snapshots__/bridge.integration.test.ts.snap

```diff
@@ -20,12 +20,12 @@ Array [
     ],
     "id": "js:2:elrond:erd1vgfp3g7azqjx4wsmtt7067m0l62v3psmqzr24j6xvywj2tlz0gesvyzsq2:",
     "index": 0,
-    "name": "Elrond 1",
+    "name": "MultiversX 1",
     "nfts": undefined,
     "operationsCount": 6,
     "pendingOperations": Array [],
     "seedIdentifier": "",
-    "spendableBalance": "175162251239346240",
+    "spendableBalance": "180162251239346240",
```

Comment:
Wondering why the balance changed here, does that indicate the account isn't frozen?

Reply:
This means that users can now spend all their balance.